slsDetectorSoftware/src/Module.cpp

```diff
@@ -1722,10 +1700,7 @@ void Module::updateReceiverStreamingIP() {
 }
 
 bool Module::getTenGigaFlowControl() {
-    int retval = -1;
-    sendToDetector(F_GET_TEN_GIGA_FLOW_CONTROL, nullptr, retval);
-    LOG(logDEBUG1) << "ten giga flow control :" << retval;
-    return retval == 1;
+    return sendToDetector<int>(F_GET_TEN_GIGA_FLOW_CONTROL)!=0;
```

Comment:
I think we can return an int, and it will cast itself into bool, right? We dont need to check if its not equal to 0

Reply:
Done